projects/canopy/src/lib/pagination/pagination.component.html

```diff
@@ -0,0 +1,40 @@
+<ul *ngIf="pages.length > 1">
+  <li>
+    <button
+      type="button"
+      class="lg-pagination__button"
+      aria-label="Go to previous page"
+      lgMarginRight="xxs"
+      (click)="previous()"
+      [disabled]="currentPage === 1"
+    >
+      <lg-icon name="chevron-left"></lg-icon>
+    </button>
+  </li>
+  <li *ngFor="let page of pages">
+    <button
+      type="button"
+      class="lg-pagination__button"
+      lgMarginRight="xxs"
+      [attr.aria-label]="'Go to page ' + page"
+      [attr.aria-current]="currentPage === page ? 'true' : null"
```

Comment:
Could this just be `[attr.aria-current]="currentPage === page`?

Reply:
TBH, i'm not entirely sure. If set wet it to page then it will set the attribute to the page number, but looking at the [Acessibility docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current), that's not a valid value. It suggest setting it to true

Reply:
![Screenshot 2023-03-02 at 16 56 17](https://user-images.githubusercontent.com/8397116/222498254-ad52403c-3b51-4c10-b5fe-2de7bbd0413d.png)

`false` seem to be also accepted. It just means that one is not the current page, right? 🤔

Reply:
@NemirStephan I wasn't suggesting to set it to page, but to set it to the boolean value of whether `currentPage === page`, e.g. true/false, which is what the docs suggest are the two accepted values :)

Reply:
Sorry, yes, was not paying attention.